frontend/modules/store.ts

```diff
@@ -54,3 +58,14 @@ export const store = configureStore();
 export const dispatch = store.dispatch;
 
 export const getState = store.getState;
+
+if (IS_DEVELOPMENT) {
```

Comment:
Do we still want this here or was it just for development?

It's fine either way, just checking!